pkg/client/unversioned/restclient.go

```diff
@@ -67,6 +69,7 @@ func NewRESTClient(baseURL *url.URL, apiVersion string, c runtime.Codec, maxQPS
 	}
 	return &RESTClient{
 		baseURL:    &base,
+		prefix:     prefix,
```

Comment:
this is probably better named `serverBase`, and we should rename baseURL to `apiBaseURL`